app/graphql/schema.graphql

```diff
@@ -102,6 +102,9 @@ type Profile implements Node & RulesPreload {
   ): String!
   majorOsVersion: String!
   name: String!
+  osMajorVersion: String!
+  osMinorVersion: String!
+  osVersion: String!
```

Comment:
I'm wondering if we need a composite `os_version`. What's the benefit of the duplicity?

Reply:
I guess parity, i.e. the same stuff defined on the model and in GraphQL. At least that's my perception as I see both `majorOsVersion` and `osMajorVersion` and they're aliased in the model. Or if I'm wrong, we're saving an operation on the frontend :laughing: :see_no_evil:

Reply:
IMO this is important for API users to see the same thing (i.e. the tab name) as UI users. For the UI implementation, it's also helpful because there need not be any logic to join major and minor versions. In general, I prefer to do any data translations in the backend and expose them directly in the API this way.

The reason I've added osMajorVersion (in addition to majorOsVersion) is because our search terms here are `os_major_version` and `os_minor_version`. It makes sense for the attributes to match the search terms exactly where possible.